### PR TITLE
dcterms property updates

### PR DESCRIPTION
**Pull request by zednis**
Proposed updates to the ontology file to use foaf:Person instances with the dcterms creator and contributor properties.

References issue #2.

### CODE REVIEW

File: gcis.ttl [hunk 1]
```diff
@@ -19,10 +19,58 @@
...
+	foaf:name "Justin Goldstein" ;
+	foaf:mbox <mailto:jgoldstein@usgcrp.gov> .
+
+gcis:linyun_fu a foaf:Person ;
+	foaf:name "Linyun Fu" ;
+	foaf:mbox <ful2@rpi.edu> .
+
+gcis:brian_duggan a foaf:Person ;
+	foaf:name "Brian Duggan" ;
+	foaf:mbox <mailto:duggan@ucar.edu> .
```
Comment (bduggan) [inline gcis.ttl]: This email address should be bduggan@usgcrp.gov.

**Comment by zednis** [comment]
@bduggan I have updated your email with a new commit.

File: gcis.ttl (hunk 1 above)
Comment (zednis) [inline gcis.ttl]: fixed in new commit.

**Comment by bduggan** [comment]
Looks good to me.

**Comment by zednis** [comment]
Ok, merging and closing this pull request.

**Pull request merged**